skycultures/greek_leidenAratea/description.en.utf8

```diff
@@ -0,0 +1,325 @@
+		<td>Crown (Latin), Wreath (Greek)</td>
+		<td> <img width="255" src="descr_pics/aratea_small_crB.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td>Hercules, Her</td>
+		<td>&#702;&Epsilon;&nu;&gamma;&#972;&nu;&alpha;&sigma;&iota;&nu;</td>
+		<td>Hercules (Latin), Kneeler (Greek) <br> The star between the legs is a well-known mistake in the picture: for this figure the draughtsman obviously took Bootes as a model; leg position, shepherd's crook (instead of club for Hercules) and star between the knees are the same.</td>
+		<td> <img width="255" src="descr_pics/aratea_small_her.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td>Lyra, Lyr</td>
+		<td>&Lambda;&#973;&rho;&alpha; </td>
+		<td>Lyre</td>
+		<td> <img width="255" src="descr_pics/aratea_small_lyr.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td>Cygnus, Cyg </td>
+		<td>&#702;&#714;&Omicron;&rho;&nu;&iota;&sigmaf; 	</td>
+		<td>Bird [not specified] </td>
+		<td> <img width="255" src="descr_pics/aratea_small_cyg.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td>Queen Cassiopeia, Cas</td>
+		<td>&Kappa;&alpha;&sigma;&sigma;&iota;&#941;&pi;&epsilon;&iota;&alpha; </td>
+		<td>Queen Kassiepeia </td>
+		<td> <img width="255" src="descr_pics/aratea_small_cas.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td>Perseus, Per	</td>
+		<td>&Pi;&epsilon;&rho;&sigma;&epsilon;&#973;&sigmaf;</td>
+		<td>Perseus <br> mirrored depiction</td>
+		<td> <img width="255" src="descr_pics/aratea_small_per.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td>Auriga, Aur		</td>
+		<td>&#703;&Eta;&nu;&#943;&omicron;&chi;&omicron;&sigmaf; </td>
+		<td>Charioteer	</td>
+		<td> <img width="255" src="descr_pics/aratea_small_aur.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td> Aquila et Sagitta,<br> Aql and Sge</td>
+		<td> &#702;&Alpha;&epsilon;&tau;&#972;&sigmaf; and &#702;&Omicron;&iota;&sigma;&tau;&#972;&sigmaf; </td>
+		<td> Eagle with Arrow </td>
+		<td> <img width="255" src="descr_pics/aratea_small_aql.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td> Delphinus, Del </td>
+		<td> &Delta;&epsilon;&lambda;&phi;&#943;&sigmaf; </td>
+		<td> Dolphin <br> In the Greek original (Aratos, ~300 BCE), there was no constellation Equuleus. The Farnese Globe (as well as the Mainz Globe and the Kugel Globe) depicts the Dolphin rahter largely, covering the space of Equ. That is why, I follow this Aratean tradition also for the Leiden Aratea. 	</td>
+		<td> <img width="255" src="descr_pics/aratea_small_del.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td> Pegasus, Peg</td>
+		<td> &#703;&#714;&Iota;&pi;&pi;&omicron;&sigmaf; </td>
+		<td> Pegasus (name of a winged horse; Latin), a Horse (Greek) </td>
+		<td> <img width="255" src="descr_pics/aratea_small_peg.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td>Andromeda, And </td>
+		<td> &#702;&Alpha;&nu;&delta;&rho;&omicron;&mu;&#941;&delta;&alpha;	</td>
+		<td> Princess Andromeda 	</td>
+		<td> <img width="355" src="descr_pics/aratea_small_and.jpg"/> 		</td>
```

Comment:
Image width of all figures in this table should be harmonized. Simple typo= (355-->255)